docs/getting-started/quickstart-install.md

```diff
@@ -25,12 +26,19 @@ Before you can get started with a Knative `quickstart` deployment you must intst
 To get started, install the Knative `quickstart` plugin:
 
 === "Using Homebrew"
-    For macOS, you can install the `quickstart` plugin by using [Homebrew](https://brew.sh){target=_blank}:
+    For macOS:
```

Comment:
I think we can remove the mac OS bit too (since homebrew can also be used on linux)